lib/folio_client.rb

```diff
@@ -120,7 +120,7 @@ def has_instance_status?(...)
 
   def data_import(...)
     DataImport
-      .new(self, ...)
-      .import
+      .new(self)
+      .import(...)
```

Comment:
Now all the instance methods in `FolioClient` use this same setup, passing the client and only the client at instantiation time.

Reply:
(It's not because everything needs to be consistent all the time, but more because there was no need to have this vary, and the pattern for new methods is now clearer.)

Reply:
whoa - is this a breaking change for consumers?    If so ... we need to maybe deal with the gem upgrade for the consumers?

Reply:
@ndushay Fortunately, there are no consumers using this method at the moment as far as I can tell after searching both GitHub and my local clones.

Reply:
Usage of the import method will need to be added to hydra_etd, but the issue for this was just unblocked and is still in the backlog: https://github.com/sul-dlss/hydra_etd/issues/1391

Reply:
(Just verifying what @mjgiarlo said about it not being in use yet.)

Reply:
Oh, and this change would only be breaking for any consumers reaching into the implementation details of the gem. It's an internal refactoring only.